lib/gems/pending/util/extensions/miq-file.rb

```diff
@@ -1,32 +1,16 @@
 require 'sys-uname'
-require 'Win32API' if Sys::Platform::OS == :windows
 require 'util/runcmd'
 require 'uri'
 require 'util/MiqSockUtil'
 require 'addressable'
 
 class File
   def self.paths_equal?(f1, f2)
-    if Sys::Platform::OS == :windows
-      # Note: The file needs to exist and be accessable for getShortFileName to work.
-      f1 = File.getShortFileName(f1).downcase
-      f2 = File.getShortFileName(f2).downcase
```

Comment:
The only callers of `getShortFileName` across the code base are these lines here, so we can remove `getShortFileName` entirely now.